src/swaps/liquality/LiqualitySwapProvider.js

```diff
@@ -164,6 +164,19 @@ class LiqualitySwapProvider extends SwapProvider {
       return mapValues(totalFees, (f) => unitToCurrency(cryptoassets[asset], f))
     }
 
+    if (txType === LiqualitySwapProvider.txTypes.SWAP_INITIATION && asset === 'NEAR') {
+      const fees = {}
+      // default storage fee recommended by NEAR dev team
+      // It leaves 0.02$ dust in the wallet on max value
+      const storageFee = BN(0.00125)
+      for (const feePrice of feePrices) {
+        fees[feePrice] = getTxFee(LiqualitySwapProvider.feeUnits[txType], asset, feePrice).plus(
```

Comment:
Shouldn't it also check that `max` is `true`?

Reply:
Nah, because we need to pay for this storage every time, just before when Max were not selected users were left with some amount of funds and they were able to cover the storage fee